src/TritonRoute/src/gc/FlexGC_main.cpp

```diff
@@ -800,10 +800,9 @@ void FlexGCWorker::Impl::checkMetalSpacing()
 {
```

Comment:
It is better to just format the files you actually worked on.  Formatting everything makes it hard to see what is really changed.